test/CreatorToken.t.sol

```diff
@@ -484,3 +497,51 @@ contract Pausing is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract bondingCurveTests is CreatorTokenTest {
+  function test_BuyPriceIsCorrect(address _buyer) public {
+    vm.assume(
+      _buyer != address(0) && _buyer != address(creatorToken) && _buyer != creator
+        && _buyer != admin
+    );
+    uint256 buyNTokens = 5;
+
+    for (uint256 i = 0; i < buyNTokens; i++) {
+      uint256 originalBuyerBalance = creatorToken.balanceOf(_buyer);
+      (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextBuyPrice();
+      uint256 _totalPrice = _tokenPrice + _creatorFee + _adminFee;
+      require(_tokenPrice == bondingCurve.priceForTokenNumber(creatorToken.totalSupply() + 1));
+      require(_totalPrice == calculateTotalPrice(_tokenPrice));
+
+      buyAToken(_buyer);
+      assertEq(creatorToken.balanceOf(_buyer), uint256(originalBuyerBalance + 1));
```

Comment:
If we have these assertions in the `buyAToken` helper (which we should!), then shouldn't need them here, I think.

Reply:
They are extra. Just felt safer that way. But we can remove it.

Reply:
Removed them here: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/6a0ad6858e103fa6da6384492f59571f85783ba2